packages/Counter/src/Counter.styles.js

```diff
@@ -69,10 +69,17 @@ const counterStyles = css`
   * {
     box-sizing: border-box;
   }
-
-  ${({ hasIndicator }) => hasIndicator && indicatorStyles};
-  ${({ color }) => counterColorStyles[color]};
-  ${({ size }) => counterSizeStyles[size]};
 `;
 
-export default counterStyles;
+export const counterStyles = styled(commonStyles)`
+  ${props => {
+    const hasIndicator = props.hasIndicator && indicatorStyles;
+    const color = counterColorStyles[props.color];
+    const size = counterSizeStyles[props.size];
+
+    return `
+      ${hasIndicator};
+      ${color};
+      ${size};`;
+  }}
+`;
```

Comment:
```suggestion
export const counterStyles = styled.span`
  ${commonStyles};
  ${props => `    
    ${props.hasIndicator && indicatorStyles}
    ${counterColorStyles[props.color]}
    ${counterSizeStyles[props.size]}
  `}
`;
```

Reply:
Maybe that would be simpler? It assumes you remove `styled(commonStyles)` from line `74` too.

Reply:
I'll want to adjust the other components to follow the same styling pattern, but it shall be done 👍